### PR TITLE
repo

### PR DESCRIPTION
**Pull request by caolinjian**
add a repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -665,7 +665,7 @@ vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https:/
 ### Projects Using Vue.js
 
 - #### Open Source
-
+  - [Rubik Cube](https://caolinjian.github.io/rubik-cube/dist/index.html) <sup>[[Source]](https://github.com/caolinjian/rubik-cube)</sup>: 魔方
```
Comment (kazupon) [inline README.md]: Can you update your commit according to the below guideline please ?

> Always add your items to the end of a list. To be fair, the order is first-come-first-serve.

https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md

File: awesome_vue_with_repo_info.md [hunk 2]
```diff
@@ -2005,6 +2005,17 @@ application.  By @Haixing-Hu
    Open Source
   </h4>
   <ul>
+  <li>
```
Comment (kazupon) [inline awesome_vue_with_repo_info.md]: In currently, so not use `awesome_vue_with_repo_info.md`

File: README.md (hunk 1 above)
Comment (caolinjian) [inline README.md]: sorry

Comment (caolinjian) [inline README.md]: sorry ****

**Pull request merged**